src/features/people/PeopleDetails/peopleDetailsSaga.js

```diff
@@ -4,14 +4,17 @@ import {
     selectPersonId,
     fetchDataError,
     getDetailsForPerson,
+    setPeopleCredits,
 } from "./peopleDetailsSlice";
-import { getPeopleDetails } from "./getPeopleDetails";
+import { getPeopleDetails, getPeopleCredits } from "./getPeopleDetails";
 
 function* fetchPersonDetailsHandler() {
     try {
         const id = yield select(selectPersonId);
         const details = yield call(getPeopleDetails, { personId: id });
+        const credits = yield call(getPeopleCredits, { personId: id });
```

Comment:
It seams like there is no need to wait for peopleData when asking for peopleCredits so You can use `yield all []` to ask for both simultaneously 🚀

Reply:
Great suggestion! Thanks for the optimization tip. I'll use yield all to request both peopleData and peopleCredits simultaneously. Appreciate your insights! 🚀👍🏻